core/templates/dev/head/components/CkEditorRteDirective.js

```diff
@@ -92,10 +92,14 @@ oppia.directive('ckEditorRte', [
         CKEDITOR.plugins.addExternal(
           'pre', '/extensions/ckeditor_plugins/pre/', 'plugin.js');
 
+        var setFocus = true;
```

Comment:
Can you name this so that it does not start with a verb?

Reply:
Done.